### PR TITLE
Erc20detailed partial test fixes

### PR DESCRIPTION
**Pull request by gtrepta**
fixes: https://github.com/runtimeverification/firefly/issues/281

### CODE REVIEW

File: tests/web3/erc20/erc20detailed-partial.in.json [hunk 1]
```diff
@@ -182,6 +194,12 @@
     false
   ]
 }
+{
+  "jsonrpc": "2.0",
+  "id": 0,
+  "method": "evm_increaseTime",
+  "params": [ 10 ]
+}
```
Comment (ehildenb) [inline tests/web3/erc20/erc20detailed-partial.in.json]: Maybe we should be increasing the time when we do the `firefly_genesisBlock`?

Comment (gtrepta) [reply tests/web3/erc20/erc20detailed-partial.in.json]: Why should we do that? We would still need `evm_increaseTime` for subsequent blocks.

Comment (ehildenb) [reply tests/web3/erc20/erc20detailed-partial.in.json]: Does `evm_mine` increase the time in Ganache? If so, we need to do the same.

Ethereum blocks come roughly 10s apart, so my guess is that Ganache is adding 10s to the time each time a block is mined.

**Comment by ehildenb** [comment]
Please also update the other two tests `tests/web3/*-partial.*`.

**Pull request merged**